util/issue_commands.go

```diff
@@ -35,9 +41,15 @@ func InitRDDLReissuanceProcess(ctx sdk.Context, proposerAddress string, tx_unsig
 func SendRDDLReissuanceResult(ctx sdk.Context, proposerAddress string, txID string, blk_height uint64) error {
 	// Construct the command
 	sending_validator_address := config.GetConfig().ValidatorAddress
+	keyring := config.GetConfig().PlanetmintKeyring
 	cmd := exec.Command("planetmint-god", "tx", "dao", "reissue-rddl-result",
 		"--from", sending_validator_address, "-y",
 		proposerAddress, txID, strconv.FormatUint(blk_height, 10))
+	if keyring != "" {
+		cmd = exec.Command("planetmint-god", "tx", "dao", "reissue-rddl-result",
```

Comment:
If I see correctly the cmd here is only different in `txID` vs. `tx_undiged` above. Can we reduce copy&pasting?

Reply:
i was mistaken. one is proposal and one is result